javalib/src/main/scala/java/io/InputStreamReader.scala

```diff
@@ -90,6 +90,8 @@ class InputStreamReader(private[this] var in: InputStream,
          */
         readMoreThroughOutBuf(cbuf, off, len)
       }
+    } else {
+      return -1
```

Comment:
Our style checker will not like this. Just write `-1` and drop `return` (which is not need here).